docs/src/content/docs/utilities/navigation-end.md

```diff
@@ -24,12 +26,34 @@ import { NavigationEnd } from '@angular/router';
 	template: '<p>Example Component</p>',
 })
 export class ExampleComponent {
-	navigationEnd$ = injectNavigationEnd();
+	source$ = injectNavigationEnd();
 	constructor() {
-		navigationEnd$.subscribe((event: NavigationEnd) => {
-			// This code will run when a navigation ends.
+		source$.subscribe((event: NavigationEnd) => {
 			console.log('Navigation ended:', event);
 		});
 	}
 }
 ```
+
+## Use Outside of an Injection Context
+
+The `injectNavigationEnd` function accepts an optional `Injector` parameter, enabling usage outside of an injection context.
+
+```ts
+@Component()
+export class ExampleComponent {
+	private readonly injector = inject(Injector);
+	source$ = injectNavigationEnd(this.injector);
```

Comment:
Shouldn't this be inside ngOnInit to show it correctly?

Reply:
done